src/main/java/fr/alasdiablo/jerintegration/compat/miningmaster/MiningMasterWorldGen.java

```diff
@@ -0,0 +1,192 @@
+package fr.alasdiablo.jerintegration.compat.miningmaster;
+
+import fr.alasdiablo.jerintegration.api.WorldGenIntegration;
+import jeresources.api.IWorldGenRegistry;
+import jeresources.api.conditionals.Conditional;
+import jeresources.api.distributions.DistributionTriangular;
+import jeresources.api.drop.LootDrop;
+import jeresources.api.restrictions.DimensionRestriction;
+import jeresources.api.restrictions.Restriction;
+import net.minecraft.world.item.ItemStack;
+import org.infernalstudios.miningmaster.init.MMBlocks;
+import org.infernalstudios.miningmaster.init.MMItems;
+import org.jetbrains.annotations.NotNull;
+
+public class MiningMasterWorldGen extends WorldGenIntegration {
+    @Override
+    public void registerWorldGen(@NotNull final IWorldGenRegistry registry) {
+        registerStoneType(registry);
+        registerDeepslateType(registry);
+
+        registry.register(
+                new ItemStack(MMBlocks.HEART_RHODONITE_ORE.get()),
+                new DistributionTriangular(1, 1, 64, 64),
```

Comment:
Mining Master did have a config system, or is hard coded ?

Reply:
They do not generate a config

In theory you could configure everything with datapacks

Maybe for the future there could be a system where you read the `PlacedFeature` resource as json or sth. to get the actual values dynamically

Reply:
Yep, my other mod Janoeo Foundation have now config file, but i need to check who to get access to those file durring runtime.